.travis.yml

```diff
@@ -1,9 +1,7 @@
 language: go
 
-sudo: required
-dist: trusty
```

Comment:
trusty is old; switch to default/current bionic.  `sudo:` is obsolete, and is always provided now.

Reply:
👍